nais/nais.yaml

```diff
@@ -36,6 +36,7 @@ spec:
     outbound:
       external:
         - host: www.nav.no
+        - host: dekoratoren.dev.nav.no
```

Comment:
Denne vil peke åpne for dekoratoren.dev.nav.no i prod også, er det tilsiktet? 
Mulig det skal være en i vars.yaml

Reply:
Kan vi sikkert :) Men er det noen risiko forbundet med å ha den åpen?

Reply:
mm, tenker at det som er fin ting at vi ikke åpner opp for ting som strengt tatt ikke er nødvendig. Trenger ikke det å være en egen host for prod?

Reply:
👍